.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Make sure you can reproduce with the latest version available**
+`pip install milatools --upgrade`
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
I would add that if there is an error traceback, please paste it here.

Reply:
Done